lib/controllers/meals-controller.js

```diff
@@ -1,7 +1,37 @@
-const Meal = require('../models/meal')
+const Meal = require('../models/meal');
 
-// const getMeal = (request, response, next) => {
-//   call a method on the Meal model
-// }
-//
-// module.exports = { getMeal }
+const index = (request, response) => {
+  Meal.allFoods()
+    .then(data => {
+      const breakfastObj = { foods: [] };
+      const lunchObj = { foods: [] };
+      const dinnerObj = { foods: [] };
+      const snackObj = { foods: [] };
+      const meals = [breakfastObj, lunchObj, dinnerObj, snackObj];
+
+      data.rows.forEach((meal) => {
+        if (meal.id === 1) {
+          createObject(breakfastObj, meal)
+        } else if (meal.id === 2) {
+
```

Comment:
@chantal66 looks like we're not creating a lunch object

Reply:
sorry i left the space blank